edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Edgelet.Docker/CombinedEdgeletConfigProvider.cs

```diff
@@ -125,18 +127,26 @@ void InjectNetworkAliases(IModule module, CreateContainerParameters createOption
 
         void MountSockets(IModule module, CreateContainerParameters createOptions)
```

Comment:
I thought we agreed that we'd remove this logic from EA entirely.

Reply:
Yes, but thinking about it we still need it even if we mount socket instead of folders.

The alternative, as we discussed was to intercept the mounting request in edgelet and do all modifications there.

I discussed with Varun too, we rather modified edgeAgent than have the whole logic inside edgelet that override the volumes.